make-nagios-commands-cfg.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+smtp_server = os.getenv('SMTP_SERVER', 'localhost')
```

Comment:
Shouldn't we fail if the SMTP server configuration is not given, instead of generating dummy config?